### PR TITLE
fix: Handle custom fees

### PR DESCRIPTION
**Pull request by AndonMitev**
## What?
Use custom fee as multiplier 

#Notion/ Trello
https://www.notion.so/Unable-to-set-custom-fees-on-uniswap-dapp-a9167540fd864fcb85537a3126a920f6

### CODE REVIEW

File: src/views/PermissionSend.vue [hunk 1]
```diff
@@ -286,8 +286,15 @@ export default {
     },
     feeInUsdValue () {
       const gas = BN(this.request.args[0].gas, 16)
-      const feePerGas = BN(this.fees[this.activeNetwork]?.[this.activeWalletId]?.[this.assetChain][this.selectedFee].fee).div(1e9)
```
Comment (bradleySuira) [inline src/views/PermissionSend.vue:289]: Hi, maybe we can add a default value if we can't get the .fee field inside this.fees

BR!

Comment (AndonMitev) [reply src/views/PermissionSend.vue:289]: Do you have any in mind, for example 21000?

Comment (bradleySuira) [reply src/views/PermissionSend.vue:289]: O no sorry for not add details, I mean when we want to access to `this.fees[this.activeNetwork]?.[this.activeWalletId]?.[this.assetChain][this.selectedFee].fee` maybe we can get null/undefined and then `BN` constructor can throw an error.

File: src/views/PermissionSend.vue [hunk 2]
```diff
@@ -210,6 +212,21 @@ export default {
         }
       }
     },
+    async estimateGas() {
```
Comment (monokh) [inline src/views/PermissionSend.vue:215]: Can you ensure that this is functioning correctly for other injections?
This permission screen i believe is shared among others so i'm wondering about something ETH specific being introduced here.

For example try a bitcoin send

some simple docs on bitcoin injection :

https://liquality.io/documentation/web_extension/bitcoin/getting_started/

File: src/views/PermissionSend.vue [hunk 3]
```diff
@@ -237,7 +254,22 @@ export default {
       } else {
         this.updateSendFees(this.amount)
       }
-    }, 800)
+    }, 800),
+    async feeInUsdValue () {
+      const gas = await this.estimateGas();
```
Comment (monokh) [inline src/views/PermissionSend.vue:259]: I think it's not good to call this again

Ideally this function should be a computed property and use the component data as much as possible
Just calculating the USD value of whatever the fee is

**Review by monokh — CHANGES_REQUESTED** [review]

File: src/views/PermissionSend.vue (hunk 2 above)
Comment (AndonMitev) [reply src/views/PermissionSend.vue]: Tested and verified, works well for BTC

File: src/views/PermissionSend.vue (hunk 3 above)
Comment (AndonMitev) [reply src/views/PermissionSend.vue]: Did it in ideally scenario :)

**Review by monokh — APPROVED** [review]

**Pull request merged**